### DISCUSSION TITLE
Phaser classes should not  bundled in SpinePlugin

### DISCUSSION THREAD

**Discussion by Pong420** [photonstorm/phaser#5909] [Feature Requests]
These Phaser classes are bundled in the output file and make the file size larger
https://github.com/photonstorm/phaser/blob/bcec70077628ca8fece4b99ef7500d2248ed42da/plugins/spine/src/SpineFile.js#L6-L14

**Comment by photonstorm** [comment]
If you (or anyone else) can figure out a way to exclude them from the build process then I'm all ears, otherwise, it's a relatively tiny price to pay given the runtime size itself is hundreds of KB.

**Comment by Pong420** [comment]
How about written in this way. 

```js
/**
 * @typedef {import('../../../types/phaser')}
 */

var Class = require('../../../src/utils/Class');
var GetFastValue = Phaser.Utils.Objects.GetFastValue;
var ImageFile = Phaser.Loader.FileTypes.ImageFile;
var IsPlainObject = Phaser.Utils.Objects.IsPlainObject;
var JSONFile = Phaser.Loader.FileTypes.JSONFile;
var MultiFile = Phaser.Loader.MultiFile;
var TextFile = Phaser.Loader.FileTypes.TextFile;
```
